packages/@glimmer/validator/lib/utils.ts

```diff
@@ -11,3 +11,18 @@ export const symbol =
   typeof Symbol !== 'undefined'
     ? Symbol
     : (key: string) => `__${key}${Math.floor(Math.random() * Date.now())}__` as any;
+
+export const symbolFor =
+  typeof Symbol !== 'undefined'
+    ? Symbol.for
+    : (key: string) => (`__GLIMMER_VALIDATOR_SYMBOL_FOR_${key}` as unknown) as symbol;
+
+export function getGlobal() {
```

Comment:
```suggestion
export function getGlobal(): any {
```

This would avoid the extra cast right?

Reply:
yeah, that's true, let's do that for now